src/components/DeleteEverything.tsx

```diff
@@ -21,16 +22,24 @@ export function deleteDb(name: string) {
 
 export function DeleteEverything() {
     async function resetNode() {
-        setConfirmLoading(true);
-        deleteDb("gossip")
-        deleteDb("wallet")
-        localStorage.clear();
-        showToast({ title: "Deleted", description: `Deleted all data` })
-        setConfirmOpen(false);
-        setConfirmLoading(false);
-        setTimeout(() => {
-            window.location.href = "/";
-        }, 3000);
+        try {
+            setConfirmLoading(true);
+            deleteDb("gossip")
+            deleteDb("wallet")
+            localStorage.clear();
```

Comment:
Instead of this could instead import an empty json object into the node manager, that would clear it properly and if we change the indexed db stuff we won't need to remember to change it here.

Reply:
oh dang that would be sweet